src/TLuaInterpreter.cpp

```diff
@@ -3208,6 +3208,56 @@ int TLuaInterpreter::setLabelClickCallback( lua_State *L )
     return 0;
 }
 
+int TLuaInterpreter::setLabelReleaseCallback( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "setLabelReleaseCallback: wrong argument type" );
+        lua_error( L );
+        return 1;
+    }
+    else
+    {
+        luaSendText = lua_tostring( L, 1 );
+    }
+    string luaName="";
+    if( ! lua_isstring( L, 2 ) )
+    {
+        lua_pushstring( L, "setLabelReleaseCallback: wrong argument type" );
+        lua_error( L );
+        return 1;
+    }
+    else
+    {
+        luaName = lua_tostring( L, 2 );
+    }
+
+    TEvent pE;
+
+    int n = lua_gettop( L );
+    for( int i=3; i<=n; i++)
+    {
+        if( lua_isnumber( L, i ) )
+        {
+            pE.mArgumentList.append( QString::number(lua_tonumber( L, i ) ) );
+            pE.mArgumentTypeList.append( ARGUMENT_TYPE_NUMBER );
+        }
+        else if( lua_isstring( L, i ) )
+        {
+            pE.mArgumentList.append( QString(lua_tostring( L, i )) );
+            pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
+        }
+    }
+
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    QString text(luaSendText.c_str());
+    QString name(luaName.c_str());
+    mudlet::self()->setLabelReleaseCallback( pHost, text, name, pE );
```

Comment:
Not sure this method exists in the `mudlet` class, and unfortunately this will need revision to the `TLabel` class to support the arguments that the `TEvent` instance the `pE` points to.  I suspect that the `TLabel` storage in Mudlet Maps may be affected by such a change as well. 😞